src/test/java/TestLoginPage.java

```diff
@@ -0,0 +1,63 @@
+import com.solvd.pages.base.LoginPageBase;
+import com.solvd.pages.base.ProductPageBase;
+import com.solvd.utilis.AuthError;
+import com.zebrunner.carina.core.AbstractTest;
+import com.zebrunner.carina.utils.R;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.assertEquals;
+
+public class TestLoginPage extends AbstractTest {
```

Comment:
please use following naming (`Test` at the end):
```suggestion
public class LoginPageTest extends AbstractTest {
```

Reply:
Done